test/modules/aggregation-layers/contour-layer/marching-squares.spec.js

```diff
@@ -407,13 +407,13 @@ test('MarchingSquares#getCode', t => {
       width: testCase.gridSize ? testCase.gridSize[0] : gridSize[0],
       height: testCase.gridSize ? testCase.gridSize[1] : gridSize[1]
     });
-    t.equals(code, testCase.code, `Code: expected: ${testCase.code}, actual: ${code}`);
+    t.equals(code, testCase.code, `Code: expected=${testCase.code}, actual=${code}`);
```

Comment:
Just curious, why are these a problem?

Reply:
> Just curious, why are these a problem?

Non-technical reason: I search for the tape generated string "expected: " to find errors in the long test logs, so prefer if the tests don't print "expected: " as it slows down my iterations...